src/main/webapp/resources/js/admin-deactivate-room-button.js

```diff
@@ -0,0 +1,22 @@
+$('.for-table tbody').on('click', '#deactivateButton', function() {
+    var dialog = $('#deactivateModal');
+    var btn = this;
+    dialog.modal('show');
+    $('#deactivateYesButton').click(function () {
+        $(btn).parents('form').submit();
+        dialog.modal("hide");
+    });
+    $('#deactivateNoButton').click(function () {
+        dialog.modal("hide");
+    });
+});
+
+$('.for-table tbody').on('click', '#activateButton', function() {
```

Comment:
No. It works on every button with this id when user press on it. Then i work with concrete button using $(this).